Project.toml

```diff
@@ -14,11 +15,11 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
 DataFrames = "0.20"
-Documenter = "0.24"
-MixedModels = "2"
+Distributions = "0.22"
+Documenter = "0.23"
 PooledArrays = "0.5"
 StaticArrays = "0.12"
-Tables = "0.2"
+Tables = "1"
```

Comment:
If we allow Tables 0.2 and 1.0, we get compatibility with the current release of MixedModels. We don't have anything else in here that depends on changed functionality.